pgrx-tests/tests/ui/elog/argument-order/detail_first.stderr

```diff
@@ -0,0 +1,10 @@
+error: The alternative invocation requires `message` to be specified first.
+  --> tests/ui/elog/argument-order/detail_first.rs:7:5
+   |
+7  | /     log!(
+8  | |         detail = "detail";
+9  | |         message = "...";
+10 | |     );
```

Comment:
Hmm. I don't understand why we would allow assignment at all if it's going to enforce a strict ordering: the main benefit of an assignment-based scheme, like with `format_args!`, is that I don't have to worry about that sort of thing!